types/smallwei__avue/crud.d.ts

```diff
@@ -22,276 +56,299 @@ interface AvueCrudColumn<T = any> extends AvueFormColumn<T> {
     /** 搜索框类型 */
     searchType?: string;
     /** 搜索框的标题宽度 */
-    searchLabelWidth?: number;
+    searchLabelWidth?: number | string;
     /** 搜索框的清除按钮 */
     searchClearable?: boolean;
     /** 搜索表单的是否多选 */
     searchMultiple?: boolean;
     /** 多选时是否将选中值按文字的形式展示 */
     searchTags?: boolean;
+    /** 搜索项标题位置	 */
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/smallwei__avue/crud.d.ts;
  Line: 66;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.